curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/65648eefbee2014f5815f4ba.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 65648eefbee2014f5815f4ba
+title: Step 16
+challengeType: 20
+dashedName: step-16
+---
+
+# --description--
+
+After your new variable, add a comment saying `Generate password`.
```

Comment:
```suggestion
Below your new variable, add a comment saying `Generate password`.
```

Reply:
I would suggest specifying if the comment should be below the variable (on the next line) or next to it (as an in-line comment).